src/selectors/WalletSelectors.js

```diff
@@ -53,10 +53,10 @@ export const convertCurrency = (state: RootState, fromCurrencyCode: string, toCu
   return convertedAmount
 }
 
-const convertCurrencyWithoutState = (exchangeRates: { [string]: number }, fromCurrencyCode: string, toCurrencyCode: string, amount: number = 1) => {
+const convertCurrencyWithoutState = (exchangeRates: { [string]: number }, fromCurrencyCode: string, toCurrencyCode: string, amount: string = '1'): string => {
```

Comment:
The goal is for these functions to only accept strings as arguments. For every argument that passes numbers, like exchangeRates, create a utility function to convert to strings before a function like convertCurrencyWithoutState is called. Then, update convertCurrencyWithoutState to use biggystring.